src/EditorFeatures/CSharpTest/Braces/AddBracesTests.cs

```diff
@@ -1,22 +1,38 @@
-﻿using System;
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.CodeStyle;
 using Microsoft.CodeAnalysis.CSharp.Diagnostics.AddBraces;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.Options;
 using Roslyn.Test.Utilities;
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.AddBraces
 {
-    public partial class AddBracesTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
+    public class AddBracesTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
     {
         internal override Tuple<DiagnosticAnalyzer, CodeFixProvider> CreateDiagnosticProviderAndFixer(Workspace workspace)
         {
             return new Tuple<DiagnosticAnalyzer, CodeFixProvider>(new CSharpAddBracesDiagnosticAnalyzer(),
                 new CSharpAddBracesCodeFixProvider());
         }
 
+        private IDictionary<OptionKey, object> AddBraces() =>
+            Options(CSharpCodeStyleOptions.UseBracesWherePossible, new CodeStyleOption<bool>(true, NotificationOption.None));
+
+        private IDictionary<OptionKey, object> Options(OptionKey option, object value)
+        {
```

Comment:
I think there is a helper for this in `AbstractCodeActionOrUserDiagnosticTest` which you're deriving from. Or I see one with a slightly different signature. Can you either use that or move this into `AbstractCodeActionOrUserDiagnosticTest` so that all code action tests can use it?